examples/asr/asr_streaming/speech_to_text_streaming_infer.py

```diff
@@ -0,0 +1,333 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can you move this file to `asr_streaming_inference/{ctc/rnnt}/speech_to_text_streaming_infer.py`

Reply:
It works for both CTC and RNNT models.